wolfcrypt/src/ecc.c

```diff
@@ -4913,35 +4913,50 @@ int wc_ecc_sign_hash_ex(const byte* in, word32 inlen, WC_RNG* rng,
     #else
         mp_int* sign_k = NULL;
     #endif
+    #if defined(WC_ECC_NONBLOCK) && defined(WC_ECC_NONBLOCK_ONLY)
+        /* perform blocking call to non-blocking function */
+        ecc_nb_ctx_t nb_ctx;
+        XMEMSET(&nb_ctx, 0, sizeof(nb_ctx));
+    #endif
     #ifndef WOLFSSL_SP_NO_256
         if (ecc_sets[key->idx].id == ECC_SECP256R1) {
-        #ifdef WC_ECC_NONBLOCK
+        #if defined(WC_ECC_NONBLOCK) || defined(WC_ECC_NONBLOCK_ONLY)
```

Comment:
Need to fix this as well.

Reply:
I cannot reproduce any build errors. This one should be okay. It `WC_ECC_NONBLOCK_ONLY` is not defined then it will go to the normal blocking case below.

Reply:
nb_ctx is not defined in the ecc structure when only WC_ECC_NONBLOCK_ONLY is defined.
./configure --enable-sp=yes,nonblock --enable-sp-math C_EXTRA_FLAGS=-D WC_ECC_NONBLOCK_ONLY